clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -502,6 +502,7 @@ export const CustomTextInput = ({
         </Flex>
         <TextInput
           {...field}
+          value={props.value ? props.value : field.value}
```

Comment:
Can this value be included in the initial values instead of through a prop? That would remove the need to update the base input field.